distributed/tests/test_asyncprocess.py

```diff
@@ -238,13 +238,9 @@ async def test_close():
 async def test_exit_callback():
     to_child = get_mp_context().Queue()
     from_child = get_mp_context().Queue()
-    evt = Event()
+    evt = asyncio.Event()
 
-    # FIXME: this breaks if changed to async def...
```

Comment:
this because because async functions are not supported `set_exit_callback` calls synchronous callbacks on the event loop thread, and this is intentional: https://github.com/dask/distributed/pull/6526/files